noir-projects/noir-protocol-circuits/crates/rollup-lib/src/tx_base/tests/private_tx_base/mod.nr

```diff
@@ -145,17 +146,21 @@ impl TestBuilder {
             tree_snapshot_diff_hints: std::mem::zeroed(),
             pre_existing_note_hashes: pad_end(pre_existing_note_hashes),
             note_hash_tree: std::mem::zeroed(),
-            pre_existing_public_data_leaves: pad_end(pre_existing_public_data_leaves),
+            pre_existing_public_data_preimages,
             nullifier_tree: std::mem::zeroed(),
             pre_existing_nullifiers,
-            fee_payer_balance_leaf_index,
+            fee_payer_preimage_index,
             public_data_tree: std::mem::zeroed(),
         };
 
-        builder.build_note_hash_tree();
-        builder.build_nullifier_tree();
+        // Insert the pre-existing note hashes at a non-zero index by default.
+        let note_hash_start_index = (MAX_NOTE_HASHES_PER_TX * 26) as Field;
```

Comment:
This looks like a hard-coded magic number that will trip someone up in a year when they try to make a change. Maybe explain why "26" is a good choice of number, at least.